### PR TITLE
Add serv-minimal test case

### PR DESCRIPTION
**Pull request by rkapuscik**
This is the test case attached to #598, adapted to running via UHDM-integration-tests framework.

### CODE REVIEW

File: tests/serv-minimal/src/servant_1.1.0/bench/servant_tb.cpp [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+    }
+  }
+  return false;
+}
+
+int main(int argc, char **argv, char **env)
+{
+  vluint64_t sample_time = 0;
+	uint32_t insn = 0;
+	uint32_t ex_pc = 0;
```
Comment (RRozak) [inline tests/serv-minimal/src/servant_1.1.0/bench/servant_tb.cpp:88]: The indentation is too long

File: tests/serv-minimal/src/servant_1.1.0/bench/servant_tb.cpp [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+      context->last_update += context->baud_t;
+      context->state=1;
+      return true;
+    }
+  }
+  return false;
+}
+
+int main(int argc, char **argv, char **env)
+{
```
Comment (RRozak) [inline tests/serv-minimal/src/servant_1.1.0/bench/servant_tb.cpp:85]: According to our convention, this bracket should be in the line above.

**Review by RRozak — CHANGES_REQUESTED** [review]

File: tests/serv-minimal/src/servant_1.1.0/bench/servant_tb.cpp (hunk 1 above)
Comment (rkapuscik) [reply tests/serv-minimal/src/servant_1.1.0/bench/servant_tb.cpp]: Fixed - replaced mixed tabs and spaces with spaces only.

File: tests/serv-minimal/src/servant_1.1.0/bench/servant_tb.cpp (hunk 2 above)
Comment (rkapuscik) [reply tests/serv-minimal/src/servant_1.1.0/bench/servant_tb.cpp]: Fixed

**Review by RRozak — APPROVED** [review]

**Pull request merged**